mmdet/models/utils/misc.py

```diff
@@ -412,7 +422,7 @@ def images_to_levels(target, num_levels):
 
     [target_img0, target_img1] -> [target_level0, target_level1, ...]
     """
-    target = torch.stack(target, 0)
+    target = stack_boxes(target, 0)
```

Comment:
`stack_boxes` is not suitable semantically when target is label not bbox.

Reply:
I consider the name `stack_boxes` is uniform with other functions.